### ISSUE TITLE
Just execute npm scripts

### ISSUE THREAD

**Issue by ray007** [electrode-io/xarc-run#20]
I'm just trying to run npm scripts from my `package.json`, but `xrun` is always complaining about missing `xrun-tasks.js`.

Looking at the documentation, calling
```
npx xrun -s i18n-mf:en i18n-mf:de
```
should work, but gives me an error:
```
[10:23:43] No xrun-tasks.js found - @xarc/run has nothing to do
    Please create xrun-tasks.js - sample: https://www.npmjs.com/package/@xarc/run#a-simple-example
```
Adding an option `-n` or `--npm` does not change anything.

What am I doing wrong?

**Comment by cmolina** [comment]
Did you define those tasks in the `xrun-tasks.js` file? I originally assumed I was going to be able to run npm scripts directly but that is not the case.

**Comment by jchip** [comment]
right, it exits if it doesn't find `xrun-tasks.js`.  If you just want to run your scripts in `package.json` then this is a bug.
creating an empty `xrun-tasks.js` avoids it.  
I will fix it.

**Comment by jchip** [comment]
fixed in 1.0.5
